### ISSUE TITLE
[BUG] Store page does not show price in the currency set for profile country

### ISSUE THREAD

**Issue by gordonfreeman01** [IsThereAnyDeal/AugmentedSteam#2090]
### Expected Behavior
Store page must show price in the currency set for my profile country irrespective of browser IP

### Actual Behavior
Store price is forcibly set to browser IP even with Price setting in augmented steam set to view in USD instead of auto.

### Steps to Reproduce the Problem

  1. Open any product page (game/DLC/bundle)
  2. Look at price of any item

### Specifications

  - Augmented Steam Version: 4.1.2
  - Browser Version: Firefox 133

**Comment by candela97** [comment]
I'm afraid that option only changes the currency used when displaying price history from ITAD, and doesn't affect your store currency. You can view the store in any supported currency **while logged out** by adding `?cc=` to the end of the url followed with a 2-letter country code, e.g. `https://store.steampowered.com/app/70/?cc=us`.

**Comment by gordonfreeman01** [comment]
> I'm afraid that option only changes the currency used when displaying price history from ITAD, and doesn't affect your store currency. You can view the store in any supported currency **while logged out** by adding `?cc=` to the end of the url followed with a 2-letter country code, e.g. `https://store.steampowered.com/app/70/?cc=us`.
> 
> If this is not what you meant, please explain more about your issue 😀

That option is interesting but not what I meant as you correctly surmised :) The issue I'm facing is something that only recently started happening with Augmented Steam enabled while logged into Steam in a browser. The currency should be based on what is set by Steam for my profile (e.g. ROW is USD) but Augmented Steam seems to be forcefully setting it to the geolocation-based currency (e.g. GBP), ignoring the store setting. The setting I described in the issue was an attempt at forcing it back to my preferred currency.

**Comment by candela97** [comment]
You're talking about ITAD pricing right? What is your store country?

**Comment by gordonfreeman01** [comment]
> You're talking about ITAD pricing right? What is your store country?

I'm talking about the price showing up on the Steam store page with Augemented Steam installed. In the Steam client it shows up correctly but in browser it's showing up as ZAR for example. Steam country set to Mauritius. I just tested it again and that behaviour seems to have stopped, very strange.

**Comment by candela97** [comment]
Yeah, as I said, we can't affect which currency Steam decides use for your **store** prices, we just read what is displayed on https://store.steampowered.com/account/ for use in ITAD price history. Unless you can show there's a difference with **only** Augmented Steam toggled on/off, this doesn't seem like an issue with us.

**Comment by gordonfreeman01** [comment]
> Yeah, as I said, we can't affect which currency Steam decides to use for your **store** prices, we just read what is displayed on https://store.steampowered.com/account/ for use in ITAD price history. Unless you can show there's a difference with Augmented Steam as your **only** installed extension toggled on/off, this doesn't seem like an issue with us.

Yeah I understand that. The issue really did seem to be Augmented Steam-related as I could consistently reproduce it when I logged the bug. If it happens again I'll retest, screenshot it and share to illustrate.
